pkg/network/vmispec/interface.go

```diff
@@ -102,7 +109,11 @@ func IsPodNetworkWithBridgeBindingInterface(networks []v1.Network, ifaces []v1.I
 	return true
 }
 
-func PopInterfaceByNetwork(statusIfaces []v1.VirtualMachineInstanceNetworkInterface, network *v1.Network) (*v1.VirtualMachineInstanceNetworkInterface, []v1.VirtualMachineInstanceNetworkInterface) {
+//nolint:gocritic
```

Comment:
Sorry, but I feel that if I let one of these to get in, it will be a precedence to other exceptions.
I would prefer to "eat" the named returned arguments over adding such a linter override.

Reply:
I've re-added the removal of `PopInterfaceByNetwork`.